pkg/multicluster/internal/k8s/apiextensions.k8s.io/v1/clients.go

```diff
@@ -0,0 +1,216 @@
+// Code generated by skv2. DO NOT EDIT.
```

Comment:
Should the v1beta1 counterpart to all of these generated files be removed?

Reply:
it doesn't really matter, it's just clients, we can have both